book-club/ClientApp/src/API/Controllers/Books.js

```diff
@@ -0,0 +1,14 @@
+import {api} from '../googleAxiosConfig'
+
+const controllerBase = '/v1/volumes'
```

Comment:
Controller base is for the C# API side to map to our Controllers. For the third party API you can just put the "v1" in the baseURL in the config and reference specific endpoints in the booksAPI methods